src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,26 +2,33 @@
 
 ## Unreleased
 
-* Breaking change: Rename package to OpenTelemetry.Instrumentation.AWSLambda
-  (remove ".Contrib") ([#593](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593))
-* Breaking change: Move public class AWSLambdaWrapper out of Implementation subnamespace
+## 1.1.0-beta2
+
+Release PR: [#590](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/590)
+
+This is the first release with the new package name `OpenTelemetry.Instrumentation.AWSLambda`.
+
+* BREAKING (API, behavior): Rename package to OpenTelemetry.Instrumentation.AWSLambda
+  (remove ".Contrib") ([#593](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593)).
+  This also affects the `ActivitySource` name (superseding [#534](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/534)).
+* BREAKING (API): Move public class AWSLambdaWrapper out of Implementation subnamespace
   ([#593](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593))
+* BREAKING (API): Rename overloads of `AWSLambdaWrapper.Trace` that take an async
+  handler to `TraceAsync`, to emphasize that they (usually) need to be awaited.
+  ([#608](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/608))
```

Comment:
I was made aware of an issue caused by #608: By naming the former overloads of Trace that took a Task-returning handler to TraceAsync, it is still possible to call Trace with a Task-returning handler. This will then cause the trace method to not await the result, ending the span probably immediately (and not reporting any exceptions). This seems like a bit of a pitfall to me and I'm currently considering reverting #608. Any thoughts?

Reply:
I guess either way has it's issues. Let's keep it how it is now (with separate Trace/TraceAsync).

Reply:
I think this is more preferable that you kept `TraceAsync`. It's in sync with [MS naming convention for async methods](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/task-asynchronous-programming-model).